themes/theme-core/src/components/Tag.styles.ts

```diff
@@ -2,10 +2,10 @@ import { ThemeComponent, cva } from '@marigold/system';
 
 export const Tag: ThemeComponent<'Tag'> = {
   tag: cva([
-    'border-border-brand bg-bg-surface flex items-center gap-2 rounded border',
+    'border-border-base bg-bg-base flex items-center gap-1 rounded border',
```

Comment:
I think they should be more distinguishable from the background.

Reply:
What token is bg-base? Should this even exist? 😮

Reply:
I extra make it more in direction of b2b theme :) and bg-base is the  opposite of bg-inverted. Our docs says: 

base: Default values for a specific category of tokens, serving as the foundational colors

Reply:
Otherwise we also have to adjust the b2b theme